server/qa/routes.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const axios = require('axios');
+const { baseUrl, authorization } = require('../server-config');
+
+const router = express.Router();
+
+router.get('/questions', (req, res) => {
+  const { productId } = req.query;
+  axios
+    .get(`${baseUrl}/qa/questions`, {
+      params: {
+        product_id: productId,
+      },
+      headers: {
+        Authorization: authorization,
+      },
+    })
+    .then((response) => {
+      const { results } = response.data;
+      res.status(200).send(results);
+    })
+    .catch(() => {
+      res.status(500).send('Unable to complete request.');
+    });
+});
+
+router.get('/questions/:question_id/answers', (req, res) => {
+  const { productId } = req.query;
+  // eslint-disable-next-line camelcase
```

Comment:
Do you think you could  rename the query param to questionId so that you can follow camel case? 

Like this:
`router.get('/questions/**:questionId**/answers`